src/Futhark/Actions.hs

```diff
@@ -146,6 +154,40 @@ runCC cpath outpath cflags_def ldflags = do
     Right (ExitSuccess, _, _) ->
       return ()
 
+runEMCC :: String -> String -> FilePath -> [String] -> [String] -> [String] -> Bool -> Bool -> FutharkM ()
+runEMCC cpath outpath classpath cflags_def ldflags expfuns rts lib = do
+  ret <-
+    liftIO $
+      runProgramWithExitCode
+        "emcc"
+        ( [cpath, "-o", outpath]
+            ++ ["-lnodefs.js"] -- "-s", "INITIAL_MEMORY=201326592"] -- ,"-s", "ALLOW_MEMORY_GROWTH=1"]
```

Comment:
What is the purpose of this comment?

Reply:
This was some convenient stuff for testing the different compiler settings. But this should not be in there.